templates/backOffice/default/hooks.html

```diff
@@ -510,7 +510,7 @@
 <script id="discover-missing" type="text/html">
     <table class="table table-striped table-condensed">
         <caption class="clearfix">
-            {intl l='Your template does not support this hooks'}
+            {intl l='Your template does not support these hooks'}
```

Comment:
Instead of changing the sentence in the html page, which implies changing all translation files, it is better to change only the English translation in the en_US.php translation file.